### ISSUE TITLE
Mtg: Discovery Kickoff: ROW Portal Update for TCP Intake

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#9165]
**Date & Time:** July 19 2022 @ 3pm
**Invited:** Darren, Samson, Chrispin, Diana, Karo

**Comment by KaroEngstrom** [comment]
Postponed

**Comment by TracyLinder** [comment]
### Meeting Info 

**Goals:**

- Define scope of work & MVP _(Just enough features to meet 'Must Haves')_
- Define stakeholders
- Resource estimates (Apps time, product, stakeholder)
- **Is this a project or an enhancement?**



----
**Agenda:**
Review existing documentation
Identify  based on the business workflow

**Comment by dianamartin** [comment]
# Diana's Notes
TCP Module Enhancements/Project

## Current Process
TCP Intake through Outlook

## Workflow
- Switch TCP intake from outlook to ROW Portal
  - Create ROW Customer Portal for TCP submittal.
- Create system features to prevent ROW from reviewing TCPs that haven't paid the review fee yet.
- New Case status: Pending Payment
- Case created as Pending Payment
- Capture COS data

**Comment by TracyLinder** [comment]
### Notes

- Stop using 'Approved' as a status or review determination. Use 'Review Completed' instead. 



Q: When is staff using outlook for attachments/reviews?
- Intake for new & revisions
- Sending marked up plans back to customer
- Sending the approved plans

Next steps - demo to stakeholders the 2nd or 3rd week of August. 


Deliver the intake

**Comment by TracyLinder** [comment]
During the discovery meeting #9165 we determined the scope of this request meets DTS's criteria for a project based approach. Introducing a public intake workflow to the TCP Review Mod will require significant updates to the existing mod, and building case management features similar to TIA. @TracyLinder will create a project index

We also determined we can move forward building a public TCP intake form, #8378.
